QuickDeployment/V2/QuickDeploymentDesktopReadme.md

```diff
@@ -0,0 +1,360 @@
+```
+
+Parameter descriptions are provided below for your convenience.
+You can also call `Get-Help C:\choco-setup\files\Set-QDEnvironment.ps1 -Full` for more information.
+
+| Parameter Name           | Description                                                                                                                                                                          |
+| :----------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `-NexusApiKey`           | The api key for your nexus installation.                                                                                                                                             |
+| `-CertificateThumbprint` | If using a thumbprint, specify it with this parameter.                                                                                                                               |
+| `-CertificateSubject`    | If using a subject name for your certificate, specify it with this parameter (**NOTE** this does not include the 'CN=' portion of the subject returned by the certificate provider). |
+| `-CertificateDnsName`    | If using a wildcard certificate this _must_ be included with either `-CertificateThumbprint` or `-CertificateSubject`.                                                               |
+| `-InternetEnabled`       | When this parameter is used, the QDE Server is configured for internet-facing clients.                                                                                               |
+
+---
+
+### 4. Select Community Packages to Internalize
+
+> :memo: **Note**
+>
+> If your environment is air-gapped or you have otherwise locked down access to the Community Repository, you will need to ensure the community repository is accessible for this part of the script to work.
+> It is completely optional, and you're more than welcome to skip this by selecting **Cancel** and only utilize the built in Jenkins tasks for internalization.
+
+When the script completes, you will be shown an Out-GridView prompt where you can select any of the most popular community packages to automatically internalize.
+Use `Shift` or `Ctrl` and click to select multiple.
+When you're happy with your selections (don't worry, you can always use the built-in Jenkins tasks to internalize additional packages at any time) select OK, or select Cancel to opt-out.
+
+This internalization will happen automatically in the background, and will generally take about a minute or two per package, depending on the size of the package.
+We recommend you leave the VM running while it's happening; you can keep tabs on its progress by opening Nexus to the ChocolateyInternal repository and checking the available packages.
```

Comment:
Add a note that this does NOT create or update the Jenkins job. Reading only this makes me think this would update the Jenkins job to automatically internalize updates to those packages as well.